src/lib/common/clockFunctions.cpp

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
These functions are now used from two different places, so I had to move them to a separate file.